prow/crier/reporters/gerrit/reporter.go

```diff
@@ -237,9 +237,6 @@ func (c *Client) Report(ctx context.Context, logger *logrus.Entry, pj *v1.ProwJo
 			}
 		}
 		for _, pjOnRevisionWithSameLabel := range mostRecentJob {
-			if pjOnRevisionWithSameLabel.Status.State == v1.AbortedState {
-				continue
```

Comment:
note that this might bring in more spam now.
Originally this was intended to skip legitimate aborted jobs. i.e. whenever a new patchset is uploaded

Reply:
This case was handled by `ShouldReport` as https://github.com/kubernetes/test-infra/blob/4f436d0b9431be68934a37799ed7069099391de2/prow/crier/reporters/gerrit/reporter.go#L198, so we should be safe removing this condition

Reply:
The race I was talking about was between:
- jobs from patchset `n` aborted when patchset `n+1` was pushed
- jobs from patchset `n+1` hasn't been triggered yet

In this race the condition of ` return patchsetNumFromPJ(pj) <= patchsetNum ` will not prevent spamming.

However, https://github.com/kubernetes/test-infra/blob/4f436d0b9431be68934a37799ed7069099391de2/prow/crier/reporters/gerrit/reporter.go#L123 does gate on aborted batches, when all jobs are aborted then no report will be generated